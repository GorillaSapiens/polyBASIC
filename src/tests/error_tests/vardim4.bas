a(4) := 3
a := 2
