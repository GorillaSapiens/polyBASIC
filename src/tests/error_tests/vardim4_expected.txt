ERROR: @2:10, ARRAY INDEX SIZE MISMATCH ❰a❱ DIM1 @1
